packages/lesswrong/server/callbacks/votingCallbacks.ts

```diff
@@ -4,44 +4,46 @@ import { voteCallbacks, VoteDocTuple } from '../../lib/voting/vote';
 import { postPublishedCallback } from '../notificationCallbacks';
 import { batchUpdateScore } from '../updateScores';
 
```

Comment:
You might want to look at this file in side-by-side view as I have refactored a bit. It shouldn't have changed the behaviour at all in the one user case though